core/src/main/resources/jenkins/model/Jenkins/legend_fr.properties

```diff
@@ -22,6 +22,10 @@
 
 grey=Le projet n''a jamais \u00e9t\u00e9 construit avant ou est d\u00e9sactiv\u00e9.
 grey_anime=Le premier build du projet est en cours.
+darkgrey=Le dernier build a \u00e9t\u00e9 abandonn\u00e9.
+darkgrey_anime=Le dernier build a \u00e9t\u00e9 abandonn\u00e9.  Un nouveau build est en cours.
```

Comment:
Is the double space on purpose in between the 2 sentences?

Reply:
No it's not. Thanks.